post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -63,7 +69,25 @@ public void postProcessMessage(
         }
     }
 
-    @Scheduled(fixedDelay = 5000)
+    @KafkaListener(topics = {"${spring.kafka.topic.datamart}"})
+    public void postProcessDatamart(
+            @Header(KafkaHeaders.RECEIVED_TOPIC) String topic,
+            @Payload String payload) {
+        try {
+            JsonNode payloadNode = objectMapper.readTree(payload);
+            logger.info("Got this payload: {} from the topic: {}", payloadNode, topic);
+
+            Datamart dmData = objectMapper.readValue(payloadNode.get(PAYLOAD).toString(), Datamart.class);
+            Map<Long, Long> dmMap = new HashMap<>();
+            dmMap.put(dmData.getPublicHealthCaseUid(), dmData.getPatientUid());
```

Comment:
Currently, we do not monitor PatientUid being 1, correct?

Reply:
@upasanapattnaik-eq For now there is no restriction, need to be added.